src/main/java/frc/robot/subsystems/Dashboard.java

```diff
@@ -0,0 +1,335 @@
+        _brOffset = driveSettings.add("BR Offset", 0.0).withPosition(2, 2).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var IntakeSetting = outerLayout.getLayout("Intake", BuiltInLayouts.kList).withPosition(1, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _IntakeInSpeed  = IntakeSetting.add("Intake In Speed", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 0).getEntry();
+        _IntakeOutSpeed = IntakeSetting.add("Intake Out Speed", 0.0).withWidget(BuiltInWidgets.kTextView).withPosition(0, 1).getEntry();
+
+        var climbSetting = outerLayout.getLayout("Climber", BuiltInLayouts.kList).withPosition(2, 0).withProperties(Map.of("label position", "LEFT")).withSize(7, 10);
+        _climberleftOffset  = climbSetting.add("Left Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberRightOffset = climbSetting.add("Right Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMinHeight   = climbSetting.add("Arm Minimum Height", 0.0).withPosition(0, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _climberMaxHeight   = climbSetting.add("Arm Maximum Height", 0.0).withPosition(2, 0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var noteSetting = outerLayout.getLayout("Notepath", BuiltInLayouts.kList).withPosition(3, 0).withSize(8, 10).withProperties(Map.of("label position", "LEFT"));
+        _NoteShootSpeed     = noteSetting.add("Note Shoot Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _PickupIntakeSpeed  = noteSetting.add("Pickup Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterIntakeSpeed = noteSetting.add("Shooter Intake Speed", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var shooterBedSetting = outerLayout.getLayout("Shooter Bed", BuiltInLayouts.kList).withPosition(4, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _bedMinimumAngle = shooterBedSetting.add("Bed Minimum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _bedMaximumAngle = shooterBedSetting.add("Bed Maximum Angle", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _ShooterOffset   = shooterBedSetting.add("Shooter Relative Offset", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+
+        var shooterFlywheelSetting = outerLayout.getLayout("Shooter Flywheel", BuiltInLayouts.kList).withPosition(5, 0).withSize(7, 10).withProperties(Map.of("label position", "LEFT"));
+        _maxFlywheelVelocity = shooterFlywheelSetting.add("Max Flywheel Velocity ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _maxFlywheelSpeed    = shooterFlywheelSetting.add("Max Flywheel Speed ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
+        _FlywheelIntakeSpeed = shooterFlywheelSetting.add("Flywheel Intake Speed ", 0.0).withWidget(BuiltInWidgets.kTextView).getEntry();
```

Comment:
1. Updated tab name
2. Updated variable names
3. I converted `_maxFlywheelVelocity` to `_flywheelVelocityRange`
4. Constants for defaults
5. Removed `withSize`

```suggestion
        var shooterFlywheelSettings = outerLayout.getLayout("Shooter Flywheel", BuiltInLayouts.kList).withPosition(5, 0).withProperties(Map.of("label position", "LEFT"));
        _flywheelVelocityRange = shooterFlywheelSettings.add("Flywheel Velocity Range", Constants.ShooterFlywheel.VELOCITY_RANGE).withWidget(BuiltInWidgets.kTextView).getEntry();
        _maxFlywheelSpeed      = shooterFlywheelSettings.add("Max Flywheel Speed", Constants.ShooterFlywheel.MAX_FLYWHEEL_SPEED).withWidget(BuiltInWidgets.kTextView).getEntry();
        _flywheelIntakeSpeed   = shooterFlywheelSettings.add("Flywheel Intake Speed", Constants.ShooterFlywheel.FLYWHEEL_INTAKE_SPEED).withWidget(BuiltInWidgets.kTextView).getEntry();
```